client/lib/preferences-helper/preference-list.js

```diff
@@ -17,31 +16,42 @@ import Preference from './preference';
 class PreferenceList extends Component {
 	render() {
 		const { preferences, translate } = this.props;
+
+		const sortedPreferenceItems = preferences
+			? Object.keys( preferences )
+					.sort()
+					.map( ( preferenceName ) => (
+						<Preference
+							key={ preferenceName }
+							name={ preferenceName }
+							value={ preferences[ preferenceName ] }
+						/>
+					) )
+			: [];
+
 		return (
-			<div>
+			<>
 				<QueryPreferences />
-				<a
-					href={ '/devdocs/client/state/preferences/README.md' }
-					title={ translate( 'Preferences' ) }
-				>
-					{ translate( 'Preferences' ) }
-				</a>
+				<a href={ '/devdocs/client/state/preferences/README.md' }>{ translate( 'Preferences' ) }</a>
```

Comment:
While we're here, can we convert the `{ 'string' }` to `"string"? Also, any reason to be removing the `title` attribute?